### PR TITLE
chore: removes actions from the backend

### PR DESCRIPTION
**Pull request by pandeymangg**
## What does this PR do?
Removes Actions from the backend
Fixes https://github.com/formbricks/internal/issues/326

## How should this be tested?

- Actions should be removed

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues
- [ ] First PR at Formbricks? [Please sign the CLA!](https://cla-assistant.io/formbricks/formbricks) Without it we wont be able to merge it 🙏

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

## Release Notes

- **New Features**
	- Simplified person creation process by removing monthly active user (MAU) limit checks.

- **Refactor**
	- Streamlined logic for managing person states, enhancing code clarity and maintainability by eliminating unnecessary error handling and imports.

- **Chores**
	- Conducted a cleanup of unused imports and related logic, resulting in a more efficient code structure.

### CODE REVIEW

**Review by jobenjada — CHANGES_REQUESTED** [review]
please also remove all references in the docs regarding Actions https://formbricks.com/docs/app-surveys/actions

File: packages/database/schema.prisma [hunk 1]
```diff
@@ -352,30 +351,12 @@ model ActionClass {
...
-model Action {
-  id            String      @id @default(cuid())
-  createdAt     DateTime    @default(now()) @map(name: "created_at")
-  actionClass   ActionClass @relation(fields: [actionClassId], references: [id], onDelete: Cascade)
-  actionClassId String
-  person        Person      @relation(fields: [personId], references: [id], onDelete: Cascade)
-  personId      String
-  /// @zod.custom(imports.ZActionProperties)
-  /// @zod.custom(imports.ZActionProperties)
-  /// [ActionProperties]
```
Comment (mattinannt) [inline packages/database/schema.prisma:371]: These types should also be removed from `packages/database/json-types.ts` & `packages/database/zod-utils.ts`

File: packages/types/segment.ts
Comment (mattinannt) [inline packages/types/segment.ts:1]: I think we can also remove the `packages/types/actions.ts` file

**Review by mattinannt — CHANGES_REQUESTED** [review]

**Review by mattinannt — APPROVED** [review]
Looks great! Thanks a lot for making the changes 😊🚀🔥
Let's get this merged!! 👏

**Pull request merged**